content/en/ensign-pricing/index.md

```diff
@@ -20,7 +20,8 @@ type: ensign-pricing
 
 Easy, no cost, low stakes way to experiment.
 
-{{< price-text text="Free Forever" >}}
+{{< price-text text="$0.00/mo" >}}
```

Comment:
Updated to match [Figma change](https://www.figma.com/file/PvL23fm6W36Zd88O1XVyIo/Beacon-DX-UI?type=design&node-id=10816-1514&mode=design&t=EAPW4bDd0NCnktTq-0).